### PR TITLE
Update docsite for 24.lts

### PR DESCRIPTION
**Pull request by oxve**
b/286653238

### CODE REVIEW

File: cobalt/site/docs/gen/cobalt/doc/cvals.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Here are examples of its usage:
 
 #### PublicCVals
 
-*   **Count.DOM.EventListeners** - The total number of EventListeners in
```
Comment (kaidokert) [inline cobalt/site/docs/gen/cobalt/doc/cvals.md:82]: Ugh. This is a breaking change to any client code that relies on them.

Comment (kaidokert) [reply cobalt/site/docs/gen/cobalt/doc/cvals.md:82]: Internally tracked at b/278899927 - i think that needs to be reopened.

**Review by kaidokert — APPROVED** [review]
Doc update is fine, but i think we need to follow up internally whether this change was needed at all.

File: cobalt/site/docs/gen/cobalt/doc/cvals.md (hunk 1 above)
Comment (oxve) [reply cobalt/site/docs/gen/cobalt/doc/cvals.md:82]: Resolving here. Following up internally

**Pull request merged**